### DISCUSSION TITLE
Which specific users has the access of Admin Dashboard?

### DISCUSSION THREAD

**Discussion by shubh-mehrotra** [nasirkhan/laravel-starter#427] [General]
First of all, thanks for the great platform. It looks really great.

I had one query, can you please help me with that.
Is it only allowing for the specific ids (1, 2, 3, 4)? If yes, where have we added that check?

**Comment by nasirkhan** [comment]
User access is managed by `Roles`. `Super Admin` is the top level and will have access to everything. There are other roles defined in the default application setting. You may create new roles and update existing ones by assigning permissions.

Except for the user with id 1, all the user roles can be changed and updated.

**Reply by shubh-mehrotra** [reply]
Thanks for the explanation!
